src/storage-blob-preview/azext_storage_blob_preview/operations/blob.py

```diff
@@ -697,3 +697,19 @@ def acquire_blob_lease(client, lease_duration=-1, **kwargs):
 def renew_blob_lease(client, **kwargs):
     client.renew(**kwargs)
     return client.id
+
+
+def query_blob(client, query_expression, input_config=None, output_config=None, result_file=None, **kwargs):
+
+    reader = client.query_blob(query_expression=query_expression, blob_format=input_config, output_format=output_config,
+                               **kwargs)
+
+    if result_file is not None:
+        with open(result_file, 'wb') as stream:
+            reader.readinto(stream)
+        stream.close()
+        return None
+
+    if input_config and input_config == 'ParquetDialect':
+        return reader.readall().decode("cp1252")
```

Comment:
Is this by design from service team to have different decode format? If no, we need to double check with service team.

Reply:
I was having issue when using the default UTF-8 decoding. This is a fix that I found online that has to do with some Windows encoding/decoding issue. I will double check with them.

Reply:
I just deleted this new decoding and it worked again. I am assuming it was because previously the parquet format did not work in other storage account regions so the decoding was failing.